classes/ActionScheduler_wpPostStore.php

```diff
@@ -265,44 +313,84 @@ public function query_actions( $query = array() ) {
 			$sql_params[] = $query['claimed'];
 		}
 
-		switch ( $query['orderby'] ) {
-			case 'hook':
-				$orderby = 'p.title';
-				break;
-			case 'group':
-				$orderby = 't.name';
-				break;
-			case 'modified':
-				$orderby = 'p.post_modified';
-				break;
-			case 'date':
-			default:
-				$orderby = 'p.post_date_gmt';
-				break;
-		}
-		if ( strtoupper($query['order']) == 'ASC' ) {
-			$order = 'ASC';
-		} else {
-			$order = 'DESC';
-		}
-		$sql .= " ORDER BY $orderby $order";
-		if ( $query['per_page'] > 0 ) {
-			$sql .= " LIMIT %d, %d";
-			$sql_params[] = $query['offset'];
-			$sql_params[] = $query['per_page'];
+		if ( 'select' === $select_or_count ) {
+			switch ( $query['orderby'] ) {
+				case 'hook':
+					$orderby = 'p.post_title';
+					break;
+				case 'group':
+					$orderby = 't.name';
+					break;
+				case 'status':
+					$orderby = 'p.post_status';
+					break;
+				case 'modified':
+					$orderby = 'p.post_modified';
+					break;
+				case 'claim_id':
+					$orderby = 'p.post_password';
+					break;
+				case 'schedule':
+				case 'date':
+				default:
+					$orderby = 'p.post_date_gmt';
+					break;
+			}
+			if ( 'ASC' === strtoupper( $query['order'] ) ) {
+				$order = 'ASC';
+			} else {
+				$order = 'DESC';
+			}
+			$sql .= " ORDER BY $orderby $order";
+			if ( $query['per_page'] > 0 ) {
+				$sql .= " LIMIT %d, %d";
+				$sql_params[] = $query['offset'];
+				$sql_params[] = $query['per_page'];
+			}
 		}
 
-		$sql = $wpdb->prepare( $sql, $sql_params );
+		return $wpdb->prepare( $sql, $sql_params );
+	}
 
-		$id = $wpdb->get_col($sql);
-		return $id;
+	/**
+	 * @param array $query
+	 * @param string $query_type Whether to select or count the results. Default, select.
+	 * @return array The IDs of actions matching the query
+	 */
+	public function query_actions( $query = array(), $query_type = 'select' ) {
+		/** @var wpdb $wpdb */
+		global $wpdb;
+
+		$sql = $this->get_query_actions_sql( $query, $query_type );
+
+		return ( 'count' === $query_type ) ? $wpdb->get_var( $sql ) : $wpdb->get_col( $sql );
```

Comment:
`$wpdb->get_var()` returns a string, not an array. This should either be converted to an array, or the docblock should be updated to indicate that it can return either type.